From 9bbfef386aa92fffd5957144ea9a4990c449b575 Mon Sep 17 00:00:00 2001
From: John Shao <johnshao@google.com>
Date: Wed, 25 Aug 2021 23:29:56 +0000
Subject: [PATCH 4/6] [Q_asb_2021-11] Add permission to start NFC activity to
 ensure it is from NFC stack

Bug: 191053931
Test: build
Change-Id: I41b5ddf464f45e68a4da6ad880cbc9b12e447ec5
(cherry picked from commit 024c62fa49aa2090daf50657c2b509b9478d8b92)
---
 AndroidManifest.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 882599810..79bd0fdda 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -482,6 +482,7 @@
 
         <activity
             android:name=".vcard.NfcImportVCardActivity"
+            android:permission="android.permission.DISPATCH_NFC_MESSAGE"
             android:configChanges="orientation|screenSize|keyboardHidden"
             android:theme="@style/BackgroundOnlyTheme">
             <intent-filter>
-- 
2.35.1

